src/main/java/com/iwaconsolti/school/controller/response/GradeRequest.java

```diff
@@ -0,0 +1,14 @@
+package com.iwaconsolti.school.controller.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class GradeRequest {
+    private final int id;
+    private int score;
+    private StudentRequest studentRequest;
```

Comment:
This can make the request complicate to use since you need to pass the full studentRequest when you only need the id